tools/run_remote_benchmark.sh

```diff
@@ -34,11 +34,33 @@ trap 'cleanup' EXIT
 SERVER_NAME=$BENCH_SERVER_NAME bash -x "${SCRIPT_PATH}/${CLOUD_NAME}/check_ssh_connection.sh"
 SERVER_NAME=$BENCH_CLIENT_NAME bash -x "${SCRIPT_PATH}/${CLOUD_NAME}/check_ssh_connection.sh"
 
+if [[ -z "${CONTAINER_MEM_LIMIT:-}" ]]; then
+  echo "CONTAINER_MEM_LIMIT is not set, run without memory limit"
 
-SERVER_CONTAINER_NAME=${SERVER_CONTAINER_NAME:-"qdrant-continuous-benchmarks"}
+  SERVER_CONTAINER_NAME=${SERVER_CONTAINER_NAME:-"qdrant-continuous-benchmarks"}
 
-bash -x "${SCRIPT_PATH}/run_server_container.sh" "$SERVER_CONTAINER_NAME"
+  bash -x "${SCRIPT_PATH}/run_server_container.sh" "$SERVER_CONTAINER_NAME"
 
-bash -x "${SCRIPT_PATH}/run_client_script.sh"
+  bash -x "${SCRIPT_PATH}/run_client_script.sh"
+
+  bash -x "${SCRIPT_PATH}/qdrant_collect_stats.sh" "$SERVER_CONTAINER_NAME"
+
+else
+  CONTAINER_MEM_LIMIT=${CONTAINER_MEM_LIMIT:-"700mb"}
```

Comment:
Won't be value be already set in `else` block? Why pass `700mb` as default here?

Reply:
Just a precaution, will remove.